src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateExecution.java

```diff
@@ -136,6 +140,10 @@ protected ExternalWorkspace run() throws Exception {
             throw new AbortException(message);
         }
 
+        ExternalWorkspaceActionImpl externalWorkspaceAction = new ExternalWorkspaceActionImpl(exws, flowNode);
+        flowNode.addAction(externalWorkspaceAction);
+        exws.setWorkspaceUrl(flowNode.getUrl() + externalWorkspaceAction.getUrlName());
```

Comment:
With this implementation I manage to reference the workspace url from the general `External Workspaces` page.
To be discussed.